CHANGELOG.md

```diff
@@ -35,6 +35,20 @@ release.
 
 ## [Unreleased]
 
+### Changed
```

Comment:
These need to be under a new header for 5.0.1 and there needs to be a new link at the bottom.